README.md

```diff
@@ -375,64 +373,53 @@ the Compact Thrift API port. You can configure the Jaeger exporter by following
 1. [Get Jaeger][jaeger-get-started].
 2. Configure the `JaegerExporter`
     - `ServiceName`: The name of your application or service.
-    - `AgengHost`: Usually `localhost` since an agent should 
-    usually be running on the same machine as your application or service.
+    - `AgengHost`: Usually `localhost` since an agent should usually be running on the same machine as your application or service.

```

Comment:
```suggestion
    - `AgentHost`: Usually `localhost` since an agent should usually be running on the same machine as your application or service.
```